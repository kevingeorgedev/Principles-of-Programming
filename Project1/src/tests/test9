a=8;b=*/a24;!b.
