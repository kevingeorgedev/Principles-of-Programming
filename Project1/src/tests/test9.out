loadI 1024 => r0
loadI 8 => r1
storeAI r1 => r0, 0
loadAI r0, 0 => r2
loadI 2 => r3
div r2, r3 => r4
loadI 4 => r5
mult r4, r5 => r6
storeAI r6 => r0, 4
outputAI r0, 4
